javascript/get_deleted_revs.js

```diff
@@ -0,0 +1,74 @@
+/*
+    get_deleted_revs.js
+
+    MediaWiki API Demos
+    Demo of `Deleted revisions:Get a list of deleted revision for Talk:Main Page` module
```

Comment:
Let's make this "Demo of `Deleted Revisions` module: Get a list of deleted revisions for a page" . Remove the page specification in your README files as well, so as to make the module description clearer :)

Reply:
I've done it in the others as well, hope it was right :)